providers/edge/src/airflow/providers/edge/plugins/templates/edge_worker_hosts.html

```diff
@@ -97,6 +100,25 @@ <h2>Edge Worker Hosts</h2>
                 {% endfor %}
               </ul>
             </td>
+            <td>
+              {%- if host.state == "idle" or host.state == "running" -%}
```

Comment:
This can be a bit simpler
```suggestion
              {%- if host.state in ["idle", "running"] -%}
```